Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -15,17 +15,29 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
 
     @IBOutlet weak var subheader: UILabel!
     @IBOutlet weak var lblWalletAddress: UILabel!
+    @IBOutlet var contentStackView: UIStackView!
     
     private var barButton: UIBarButtonItem?
-    
+    private lazy var syncInProgressLabel: UILabel = {
```

Comment:
Why is this preferred to including the label in Inspector Builder and creating an `IBOutlet`? Doing so will prevent needing to set constraints for this view in Swift code.

Reply:
I consider it a complimentary view. That way we maintain the storyboard clean of the main content.